lib/wda/xcodebuild.js

```diff
@@ -65,6 +65,24 @@ class XcodeBuild {
     }
   }
 
+  async retrieveDerivedDataPath () {
+    if (!this._derivedDataPath) {
+      // https://regex101.com/r/PqmX8I/1
+      const folderRegexp = /(.+\/DerivedData\/WebDriverAgent-[^\/]+)/;
+      const pid = await getPidUsingAppName(this.device.udid, 'xcodebuild');
+      if (!pid) {
+        return;
+      }
+      const {stdout} = await exec('bash', ['-c', `lsof -p ${pid} | awk '{print $9}'`]);
+      const match = folderRegexp.exec(stdout);
+      if (!match) {
```

Comment:
What did we get in `stdout` that made the regexp not work? This would help in debugging.